### PR TITLE
Add sanity test semi-automation workflow

### PR DESCRIPTION
**Pull request by h2floh**
Adding workflow and scripts to do a partial automation of the sanity-test.

We still need to create the GHES instance and a site_admin PAT token.

Workflow will download and run the sanity check the 64bit binaries for linux, windows and macos.

Added documentation on how to do a release and execute the sanity test for the new release.

### CODE REVIEW

File: docs/RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+git tag
+```
+- Push the newly created tag
+```code
+git push origin <tag>
+```
+- Check workflow `releases.yml` should trigger
+- Once completed, go to repo releases page and edit the newly created release as `pre-release`, so we can do sanity testig before we officicaly release
+- Recommend to do basic sanity testing (see below) on the new release.
+- Once sanity testing is done, we can edit the releas and mark it as `Latest version` and edit the release notes.
```
Comment (nicholasbergesen) [inline docs/RELEASE.md:23]:
```suggestion
- Once sanity testing is done, we can edit the release and mark it as `Latest version` and edit the release notes.
```

File: docs/RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- Get the tag name
+```code
+git tag
+```
+- Push the newly created tag
+```code
+git push origin <tag>
+```
+- Check workflow `releases.yml` should trigger
+- Once completed, go to repo releases page and edit the newly created release as `pre-release`, so we can do sanity testig before we officicaly release
```
Comment (nicholasbergesen) [inline docs/RELEASE.md:21]: Could you link to the repo releases page for convenience.

File: docs/RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+1. Access to a GHES test server
+1. Create a PAT token with `site-admin` scope in the GHES environment for `ghe-admin`
+
+### Execution
+
+1. Update below Repository level secrets:
+
+    - sanity_test_site_admin_token: The PAT generated earlier 
+    - sanity_test_ghes_url: The URL to the GHES instance 
+    - sanity_test_releasedatetime: The datetime string for the release to test without the `v` (e.g. `202211070205`)
```
Comment (nicholasbergesen) [inline docs/RELEASE.md:38]:
```suggestion
    - sanity_test_releasedatetime: The tag datetime string for the release to test without the `v` (e.g. `202211070205`)
```

File: docs/RELEASE.md (hunk 1 above)
Comment (h2floh) [reply docs/RELEASE.md]: fixed

File: docs/RELEASE.md (hunk 2 above)
Comment (h2floh) [reply docs/RELEASE.md]: fixed and also added link to release workflow

File: docs/RELEASE.md (hunk 3 above)
Comment (h2floh) [reply docs/RELEASE.md]: fixed

**Review by nicholasbergesen — APPROVED** [review]

**Pull request merged**